docs/API.md

```diff
@@ -32,24 +32,26 @@ protocol stream.
     [`MouseEvent.button`](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button).
     Is set to `1` by default.
 
-`viewportScale`
-  - Is a `double` indicating how the framebuffer contents should be
-    scaled before being rendered on to the canvas. See also
-    [`RFB.autoscale()`](#rfbautoscale). Is set to `1.0` by default.
-
 `clipViewport`
-  - Is a `boolean` indicating if the canvas should be clipped to its
-    container. When disabled the container must be able to handle the
+  - Is a `boolean` indicating if the session should be clipped to its
+    container. When disabled scrollbars will be shown to handle the
     resulting overflow. Disabled by default.
 
 `dragViewport`
   - Is a `boolean` indicating if mouse events should control the
-    relative position of a clipped canvas. Only relevant if
+    relative position of a clipped session. Only relevant if
     `clipViewport` is enabled. Disabled by default.
 
-`isClipped` *Read only*
-  - Is a `boolean` indicating if the framebuffer is larger than the
-    current canvas, i.e. it is being clipped.
+`scaleViewport`
+  - Is a `boolean` indicating if the session should be scaled locally
```

Comment:
I'm not sure if 'session' is a good term in general, do you think 'remote' works? This would mean changes to a number of places.

Reply:
Just "remote"? Or do you mean "remote session"?

Reply:
I was thinking only "remote", but "remote session" could work too. What phrase are you using in TigerVNC for example?

Reply:
TigerVNC uses "remote session".

Reply:
Ok, good enough for me